calcom/cal.com
# fix: API /teams/:id/publish

## What does this PR do?

Fixes API build failure introduced after #9291


## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?



- [x] Locally or on preview, check that build succeeds
- [ ] Test the [publish endpoint](https://www.postman.com/calcom/workspace/cal-com-workspace/request/20666831-c71e7234-fdc0-4f94-847a-a36bcb1434eb) against a valid api key
  - Locally, run `yarn dev:api`
  - Login to webapp and create an api key localhost:3000/settings/developer/api-keys
  - Copy the api key into the postman environment variable
  - On Postman select the local environment and overwrite the api_key with your own: 
<img width="244" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/56067b9f-866a-443a-a1f5-f946d72d3e9a">
<img width="730" alt="image" src="https://github.com/calcom/cal.com/assets/3504472/89417813-4283-415a-920a-aade35ddd551">

  - Test the endpoint with valid body data as in the postman example

## Mandatory Tasks
